pkg/handlers/redirect.go

```diff
@@ -48,18 +54,23 @@ func HandleHeaderRedirect(w http.ResponseWriter, r *http.Request, opts map[strin
 	w.Write([]byte(`User Unauthorized`))
 }
 
-func redirectParameters(w http.ResponseWriter, r *http.Request, opts map[string]interface{}) (authURLPath, sep, redirectParameter, redirectURL string, redirect bool) {
+func redirectParameters(_ http.ResponseWriter, r *http.Request, opts map[string]interface{}) (authURLPath, sep, redirectParameter, redirectURL string, loginHint string, redirect bool) {
```

Comment:
@PedroDBFlores , no need to change the signature. The `login_hint`, if any would be in `opts`.